yb-voyager/cmd/assessMigrationCommand.go

```diff
@@ -57,6 +57,9 @@ var (
 	referenceOrTablePartitionPresent = false
 	pgssEnabledForAssessment         = false
 	invokedByExportSchema            utils.BoolStr
+	sourceDBReplicaEndpoints         string
+	validatedReplicaEndpoints        []srcdb.ReplicaEndpoint // Replicas to include in assessment
+	failedReplicaNodes               []string                // Replicas that failed during metadata collection
```

Comment:
I think moving forward, we should avoid adding global variables as much as possible, and see if you can pass on the details wherever required and get rid of global state